src/Evolution/Systems/Cce/BoundaryData.hpp

```diff
@@ -77,6 +75,102 @@ void cartesian_to_spherical_coordinates_and_jacobians(
     const Scalar<DataVector>& sin_phi, const Scalar<DataVector>& sin_theta,
     double extraction_radius) noexcept;
 
+/*
+ * \brief Compute \f$g_{i j}\f$, \f$g^{i j}\f$, \f$\partial_i g_{j k}\f$, and
+ * \f$\partial_t g_{i j}\f$ from input (potentially modal Spherepack) spatial
```

Comment:
what does `potentially modal Spherepack` mean?  How is it determined whether it is true or not?